src/js/modules/core/components/ApplyPage.tsx

```
@@ -211,6 +217,47 @@ class ApplyPage extends React.Component<Props, ApplyPageState> {
                         component="input"
                       />
                     </label>
+                    <label>
+                      Gender:
+                      <Field
+                        className={classes.input}
+                        name="gender"
+                        component="select"
+                      >
+                        <option value="default"> Choose One </option>
```

Not sure we should have this as a default. Too easy for someone to submit.

---

(unless you validate that it cannot be this option)

---

I was planning on making a validation component, unless there already is one that I haven't noticed.

---

Yeah, Form takes a validate prop where you can return an errors object. Check out LoginPage or RegisterPage for an example